ddt.py

```diff
@@ -118,14 +120,18 @@ def mk_test_name(name, value, index=0):
     only of trivial values.
     """
 
+    # Add zeros before index to keep order
+    index = str(index + 1)
+    if len(index) < I_LEN:
+        index = '0' * (I_LEN - len(index)) + index
```

Comment:
There are nicer ways to let python do the zero-filling for you e.g. a couple answers here: https://stackoverflow.com/a/135157/166761